### PR TITLE
[UI] Side

### PR DESCRIPTION
**Pull request by mikelfcosta**
- Finished up the last component.
- Adjusted the board with everything.
- Board Design is basically done.

### CODE REVIEW

File: src/views/GameRouter/components/GameTable/index.tsx [hunk 1]
```diff
@@ -1,47 +1,25 @@
...
-
   return (
     <TableWrapper>
       <ActionArea
         currentCard={currentCard}
         onDeckClick={() => console.log('Deck')}
         onCurrentClick={() => console.log('Current')}
       />
-      {renderPlayers()}
+      {Object.values(players).map((player, index) => (
```
Comment (davidpn11) [inline src/views/GameRouter/components/GameTable/index.tsx:20]: SHAAAAAME

Comment (davidpn11) [reply src/views/GameRouter/components/GameTable/index.tsx:20]: Where is your fp-ts skilzz

File: src/views/GameRouter/index.tsx [hunk 2]
```diff
@@ -31,8 +32,13 @@ export default function GameRouter() {
       return (
         <GameEngine>
           <GameWrapper>
-            <GameTable />
-            <PlayerHand pass="CANNOT_PASS" solo="CANNOT_SOLO" cards={player.hand} />
+            <Main>
+              <BoardWrapper>
+                <GameTable />
+              </BoardWrapper>
+              <PlayerHand pass="CANNOT_PASS" solo="CANNOT_SOLO" cards={player.hand} />
```
Comment (davidpn11) [inline src/views/GameRouter/index.tsx:40]: Put some comment that these will be provided by the gameEngine just for visibility

**Review by davidpn11 — APPROVED** [review]

File: src/views/GameRouter/components/GameTable/index.tsx (hunk 1 above)
Comment (davidpn11) [reply src/views/GameRouter/components/GameTable/index.tsx:20]: Will we add position to session players?

Comment (mikelfcosta) [reply src/views/GameRouter/components/GameTable/index.tsx:20]: Next steps

**Pull request merged**